charts/ceph-csi-rbd/values.yaml

```diff
@@ -279,6 +279,23 @@ topology:
     - failure-domain/region
     - failure-domain/zone
 
+readAffinity:
+  # Enable read affinity for RBD volumes. Recommended to
+  # set to true if running kernel 5.8 or newer.
+  enabled: false
+  # Define which node labels to use as CRUSH location. 
+  # This should correspond to the values set in the CRUSH map.
+  crushLocationLabels:
+    - topology.kubernetes.io/region
+    - topology.kubernetes.io/zone
+    - topology.rook.io/datacenter
+    - topology.rook.io/room
+    - topology.rook.io/pod
+    - topology.rook.io/pdu
+    - topology.rook.io/row
+    - topology.rook.io/rack
+    - topology.rook.io/chassis
```

Comment:
We can just have the above labels as examples.
These rook specific labels can be removed.